### ISSUE TITLE
Java 14 switch with exception throw

### ISSUE THREAD

**Issue by youurayy** [google/google-java-format#492]
When the new switch contains an exception throw, the formatter fails with:

`Caused by: com.google.googlejavaformat.java.FormatterException: 13:30: error: expected token: '}'; generated ; instead`

```java
public class EnumBug {

  enum Sample {
    ONE,
    TWO
  }

  int func(Sample val) {
    return switch (val) {
      case ONE -> 1;
      case TWO -> throw new RuntimeException();
    };
  }
}
```

Tested with 1.8 through Gradle `com.diffplug.spotless:spotless-plugin-gradle:4.0.1`.

**Comment by jbduncan** [comment]
Hi @youurayy - what happens if you follow the instructions over at https://github.com/diffplug/spotless/tree/master/plugin-gradle#applying-to-java-source-with-google-java-format to use google-java-format 1.8, something like the following code snippet?

```gradle
spotless {
  java {
    googleJavaFormat("1.8")
  }
}
```

If this works, then I believe this bug is already fixed! Otherwise, I'm not sure yet if it's a problem with Spotless or google-java-format.

**Comment by youurayy** [comment]
Hi @jbduncan, thanks for the reply. Yes, that's how I use it:
```
spotless {
  java {
    googleJavaFormat('1.8')
  }
}
```
also tried:
```
spotless {
  java {
    googleJavaFormat('1.8').style('GOOGLE')
  }
}
```

Attaching a full stack trace:
[stack.txt](https://github.com/google/google-java-format/files/4712130/stack.txt)

`gradle -version` output:
```
------------------------------------------------------------
Gradle 6.3
------------------------------------------------------------

Build time:   2020-03-24 19:52:07 UTC
Revision:     bacd40b727b0130eeac8855ae3f9fd9a0b207c60

Kotlin:       1.3.70
Groovy:       2.5.10
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          14.0.1 (AdoptOpenJDK 14.0.1+7)
OS:           Windows 10 10.0 amd64
```

**Comment by zbychpo** [comment]
Example file:
`public class TestService  {

  protected String map( String en) {
      return switch (en) {
        case "0" -> "aaaaaa";
        case "1", "2", "3", "4" -> "bbbb";
        default -> throw new IllegalArgumentException("Unexpected value: " );
      };
  }
}

`
command:
java -jar google-java-format-1.8-all-deps.jar TestService.java

error:
TestService.java:7:56: error: expected token: '}'; generated ; instead

**Comment by jbduncan** [comment]
Cheers for confirming, @youurayy! I think @zbychpo's message above confirms that this is a problem with google-java-format, not Spotless, but as I'm not a member of the google-java-format team, or even a Google employee, the best thing we can do is wait until the team acknowledges and responds to this issue. :)

**Comment by MireskaWillow** [comment]
[I think this is a duplicate of 477](https://github.com/google/google-java-format/issues/477). If so, it's patched at head.

**Comment by youurayy** [comment]
thanks, closing

**Comment by youurayy** [comment]
FYI fix still not available (intellij plugin 1.8.0.1, gradle spotless plugin 5.1.2). Any ETA pls?
